utils/python/flash.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import print_function
+
+import argparse
+import binascii
+import ctypes
+import os
+import struct
+import sys
+from subprocess import check_output
+import time
```

Comment:
I don't think all these imports are used in this file.

Reply:
You're right; I'll remove those.

Reply:
a6b4c02